packages/mjml-cli/src/client.js

```diff
@@ -179,7 +179,7 @@ export default async () => {
   }
 
   switch (outputOpt) {
-    case 'o':
+    case "o": {
```

Comment:
I think it should be `'o'` for consistency ?

Reply:
It could be both. But best pratice tell for '' i will change it